bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/api/HydrawiseAuthenticationException.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hydrawise.internal.api;
+
+/**
+ *
+ * @author Dan Cunningham - Initial contribution
+ */
+@SuppressWarnings("serial")
+public class HydrawiseAuthenticationException extends Exception {
+
+    public HydrawiseAuthenticationException() {
```

Comment:
Is there no sensible message or cause to pass along? Then just remove this constructor, it would be generated like this anyway, but less code.